site/content/docs/docker/about.md

```diff
@@ -96,6 +96,25 @@ For more details see the blog posts:
 
 These GitHub actions are a simple way to run the packaged scans, especially if you already use GitHub.
 
+### Mounting the current directory
```

Comment:
- `### Mounting the Current Directory`
- IMO better have this as first entry in this section rather in the middle (and use the anchor in the AF entry).

Reply:
I thought I _did_ use the anchor - am I missing something?

Reply:
"See examples above." is where the anchor would be.

Reply:
Right. Done.

Reply:
The first bullet was not addressed yet.

Reply:
A bit cryptic comment, but if you mean Capitalizing the header, sure, by all means. It is not consistent in that document though...